recipe/meta.yaml

```
@@ -1,76 +1,57 @@
 {% set name = "keras" %}
-{% set version = "3.6.0" %}
+{% set version = "3.11.2" %}
 
 package:
-  name: keras
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/keras-team/keras/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 474cc5755054a1dbc71c87655c001eb411ce149988e14b1a5dae375d35e0457d
+  url: https://github.com/{{ name }}-team/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cc23944e30a7159777cce52e6fa9d6c793607ffbc3cfc281dbd20791321a2558
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -v
-  skip: true  # [py<39]
+  skip: true  # [py<310]
 
 requirements:
   host:
     - python
     - pip
-    - wheel
-    - setuptools
+    - setuptools >=61.0
   run:
     - python
     - absl-py
     - numpy
     - rich
-    - namex
+    - namex >=0.0.8
     - h5py
     - optree
     - ml_dtypes
     - packaging
   run_constrained:
-    # https://github.com/keras-team/keras/blob/v{{ version }}/requirements.txt
-    - tensorflow >=2.17.0,<3.0a
-    # pytorch should have >=2.1.0,<2.3.0 (GitHub issue #19602)
-    # a simple >=2.1.0 lower bound is used with a patch.
-    - pytorch >=2.1.0
-    - torchvision >=0.16.0
+    # https://github.com/keras-team/keras/blob/v3.11.2/requirements.txt
+    - tensorflow >=2.18.1,<2.19.0 # [py<313]
+    - tensorflow-cpu >=2.18.1,<2.19.0  # [not osx and py<313]
```

```suggestion
    - tensorflow-base >=2.18.1,<2.19.0
```
tensorflow-base covers all installation of tensorflow in conda land. These are run_constrained and will not be pulled by default when installing keras. If we set the selector py<313, a later version of tensorflow built for 3.13 would be installable with this version of keras.